validator/sawtooth_validator/server/cli.py

```diff
@@ -259,6 +259,11 @@ def main(args=sys.argv[1:]):
     if "tcp://" not in bind_component:
         bind_component = "tcp://" + bind_component
 
+    if validator_config.network_public_key is None or \
+            validator_config.network_private_key is None:
+        LOGGER.warning("Server key pair is not configured, The network is not "
+                       "secure.")
```

Comment:
Instead of "The network is not secure", let's say something like "Network communications between validators will not be authenticated or encrypted."